charts/avatar/values.yaml

```diff
@@ -19,7 +19,7 @@ serviceAccount:
     eks.amazonaws.com/role-arn: 'arn:aws:iam::602373492013:role/avatar-creation-dev'
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
-  name: avatar-creation-dev
+  name: avatar-creation
```

Comment:
The service account should be the same name as the chart. I'll change the role name to reflect the service account after you make that change.